.circleci/config.yml

```diff
@@ -19,10 +19,13 @@ jobs:
           key: jars-{{ checksum "build.gradle" }}-{{ checksum  "app/build.gradle" }}
       - run:
           name: Run Tests
-          command: ./gradlew lintPlayStoreDebug androidCheck androidPmd --no-daemon
+          command: ./gradlew lintPlayStoreDebug androidCheck androidPmd --no-daemon --max-workers 2
+      - run:
+          name: Spotless Test
+          command: ./gradlew spotlessJavaCheck
```

Comment:
This should be checked first because it doesn't need any compilation

Reply:
On it